docker/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM alpine:latest as build
+
+ARG PKG_MANAGER="npm"
+ARG INSTALL_COMMAND="npm install --production"
+
+RUN mkdir /opt/armadietto
+WORKDIR /opt/armadietto
+
+RUN apk add nodejs $PKG_MANAGER
+
+COPY package.json ./
+COPY package-lock.json ./
+
+RUN $INSTALL_COMMAND
+
+FROM alpine:latest
+
+LABEL description="Armadietto Node.js web service (a remoteStorage server)"
+
+ARG CONFIG_PATH_STORAGE="/usr/share/armadietto"
+ARG CONFIG_PATH_LOGS="/opt/armadietto/logs"
+ARG PROJECT_NAME="armadietto"
+ARG PORT="8000"
+ARG USER="armadietto"
+
+ENV NODE_ENV=production
+ENV PROJECT_NAME=$PROJECT_NAME
+ENV PORT=$PORT
+
+RUN mkdir /opt/armadietto
+WORKDIR /opt/armadietto
+
+
+RUN apk add nodejs
+
+RUN adduser -u 6582 -HD $PROJECT_NAME
+
+RUN mkdir -m 0700 $CONFIG_PATH_STORAGE
+RUN mkdir -m 0700 $CONFIG_PATH_LOGS
+RUN chown $PROJECT_NAME $CONFIG_PATH_STORAGE
+RUN chown $PROJECT_NAME $CONFIG_PATH_LOGS
+
+COPY --from=build /opt/armadietto/node_modules/ node_modules/
+COPY package.json ./
+COPY README.md ./
+COPY lib/ lib/
+COPY bin/ bin/
+
+RUN ln -s /opt/armadietto/bin/armadietto.js /usr/local/bin/armadietto
+
+COPY docker/config.json /etc/armadietto.conf.json
```

Comment:
Setting this up in k8s and I find myself changing `/etc` to `/usr/local/etc` to make it easier to mount my own config without clobbering the rest of the `/etc` folder:  data file mount.

Maybe we can keep this as `/usr/local/etc/..`?

Reply:
Yeah, I agree with the not clobbering */etc*. I did it based on the README, which has changed to */etc/armadietto/conf* now (so should update it in this...?). With the docker image, you can store the conf file wherever you want, it just needs to be mounted in */etc*, so it isn't too bad the way it is atm?